MFVNeutralino/test/jsons/ana_2017_1pc.json

```diff
@@ -1,337 +1,337 @@
-{"297050": [[264, 264], [267, 267], [312, 312], [481, 481], [511, 511]],
```

Comment:
Just to check (in general) for these 10pc and 1pc files, these are the outputs from pick10pc.py, right? (To be sure that we don't accidentally use 100% in place of 1% in the future 😀 )

Reply:
Yes I just made all those json files according to the README by running those scripts provided.